src/Ledger/Ratify.lagda

```diff
@@ -114,14 +200,29 @@ restrictedDists coins rank dists = dists -- record dists { drepStakeDistr = rest
         -- one always includes the other
         restrict : Credential ⇀ Coin → Credential ⇀ Coin
         restrict dist = topNDRepDist rank dist ∪ᵐˡ mostStakeDRepDist dist coins
-
+\end{code}
+\begin{figure*}[h]
+{\small
+\begin{code}
 CCData : Set
```

Comment:
Good point.  I moved it up so that it appears in the figure along with `StakeDistrs`, `RatifyEnv`, and `RatifyStake` and I added it to the three item list describing their definitions.  Let me know if you think that's a bad place for it and/or if/how you think the following description could be improved:

- \CCData stores data about the constitutional committee.